hail/python/hailtop/aiogoogle/client/compute_client.py

```diff
@@ -17,7 +20,7 @@ async def request_with_wait_for_done(request_f, path, params: MutableMapping[str
 
     delay = 0.2
     while True:
-        resp = await request_f(path, params=params, **kwargs)
+        resp = await retry_transient_errors(request_f, path, params=params, **kwargs)
```

Comment:
I don't think this is necessary, Cotton has retry by default in `aiogoogle.auth.session.Session.request`. BaseClient uses a Session by default. BaseClient's post, etc. all use Session's post, etc. Session's post uses request.